bin/git-imerge

```diff
@@ -0,0 +1,8 @@
+#! /usr/bin/env python
+
+from gitimerge.gitimerge import main
+import sys
+
+main(sys.argv[1:])
```

Comment:
IMO, this file shouldn't exist and instead be built using the setuptools `console_scripts` entry point feature. It is more compatible and tested across platforms.

https://python-packaging.readthedocs.io/en/latest/command-line-scripts.html#the-console-scripts-entry-point